code/emoji.py

```diff
@@ -0,0 +1,76 @@
+from talon import Module, Context
```

Comment:
I'd be tempted to make these lists into csvs.  WDYT @knausj85 @rntz ?

Reply:
I'd vote to put this into a CSV as well

Reply:
Like that?